Source/WebInspectorUI/UserInterface/Views/ComputedStyleSection.js

```diff
@@ -43,10 +46,14 @@ WI.ComputedStyleSection = class ComputedStyleSection extends WI.View
         this._propertyVisibilityMode = WI.ComputedStyleSection.PropertyVisibilityMode.ShowAll;
         this._hideFilterNonMatchingProperties = false;
         this._filterText = null;
+        this._filterCleared = false;
+        this._variablesGroupType = variablesGroupType;
```

Comment:
Style: since this is optional, we should have a `|| null` or something to make sure we don't end up with an `undefined`